lib/hq/graphql/record_loader.rb

```diff
@@ -4,6 +4,7 @@ module HQ
   module GraphQL
     class RecordLoader < ::GraphQL::Batch::Loader
       def initialize(model, column: model.primary_key, where: nil)
+        super()
```

Comment:
Why does `super` need called?

Reply:
I just disabled new cops in our default rubocop config so this shouldn't be an issue anymore. Plesae remove.